src/patternfly/components/Timestamp/timestamp.hbs

```diff
@@ -0,0 +1,6 @@
+<span class="pf-c-timestamp{{#if timestamp--modifier}} {{timestamp--modifier}}{{/if}}"
```

Comment:
@mattnolting yeah @blfetche and I chatted about that and how we take our best stab at the intended structure and how we weigh potentially bloated markup versus being limited in updating the markup later as we have design changes that may introduce a breaking change. We looked around at other design system designs and saw some icons and things next to the timestamp, so figured it's fair we might put stuff beside it. That said, I don't feel strongly about it one way or another - hopefully we'll iron out the future design requirements while it's beta. I suppose if we put something visual like an icon, it could still go in `<time>` and we just hide it, though then we may want to nest an inline-flex layout or make `<time>` an inline-flex layout.

wdyt @mcarrano @mmenestr, do you ever see putting other stuff in the timestamp component around the timestamp - beside, above, below - like text or a clock icon or actions or something? Or think it would always just be the timestamp?

Reply:
That's possible. We had some earlier designs with an icon that we abandoned, but it's always possible we'll get a request to extend this.